heron/schedulers/src/java/com/twitter/heron/scheduler/aurora/AuroraScheduler.java

```diff
@@ -109,6 +118,28 @@ public boolean onRestart(Scheduler.RestartTopologyRequest request) {
     return controller.restartJob(containerId);
   }
 
+  @Override
+  public boolean onUpdate(Scheduler.UpdateTopologyRequest request) {
+    try {
+      updateTopologyManager.updateTopology(
```

Comment:
Yes. See the changes to the ```heron_executor.py``` in this branch. On startup instead of doing:
```
self.update_instance_distribution(self.parse_instance_distribution(args[5]))
```
we do:
```
executor.register_packing_plan_watcher(executor)
```
which ignores the passed param for both submit/startup and update.